frontend/src/concepts/modelRegistry/hooks/useModelVersionTuningData.ts

```diff
@@ -39,15 +31,15 @@ export const useModelVersionTuningData = (
 
   return {
     tuningData:
-      modelVersionId && modelVersion && inputModelLocationUri
+      modelVersion && registeredModel
         ? {
             modelRegistryName: registryService?.metadata.name || '',
             modelRegistryDisplayName,
             registeredModelId: modelVersion.registeredModelId,
-            registeredModelName: modelVersion.name,
+            registeredModelName: registeredModel.name,
             modelVersionId: modelVersion.id,
             modelVersionName: modelVersion.name,
-            inputModelLocationUri,
+            inputModelLocationUri: inputModelLocationUri || '',
```

Comment:
Why the change here? Ilab won't work if we don't pass a URI. I think it was correct before where we should only return tuningData if it has what we need. Same with registryService actually, let's make sure everything is defined before we return anything here.